alipay_mobile/__init__.py

```diff
@@ -60,6 +60,11 @@ def extract(self, file, existing_entries=None):
                 units = amount.Amount(D(amt), "CNY")
                 metadata["serial"] = serial
 
+                if "自动转入" in narration:  # skip internal transfers
```

Comment:
Please do not just ignore them:

1. "自动转入" is a transaction from 余额宝 to 余额, so that would be two different accounts.
2. Although "交易关闭" marks unsuccessful transactions, they might still be useful to users, e.g. as a note. I do append a `needs-confirmation` tag to these transactions below. Maybe you can find a more elegant way of handling this.